Builds/CMake/RippledCore.cmake

```diff
@@ -153,6 +153,7 @@ install (
     src/ripple/basics/Blob.h
     src/ripple/basics/Buffer.h
     src/ripple/basics/CountedObject.h
+    src/ripple/basics/Expected.h
```

Comment:
note: we forgot to add this to the cmake headers - doesn't affect us since we build in our codebase, but others depending on libxrpl would run into this.

this makes it possible for others to depend on libxrpl